src/osd/osd_types.h

```diff
@@ -2249,6 +2249,8 @@ struct pg_stat_t {
   bool pin_stats_invalid:1;
   bool manifest_stats_invalid:1;
 
+  double scrub_duration;
```

Comment:
(while this PR is already approved, still:)
Adding this 'double' after a set of bit-fields is wasteful. Is there a requirement to add the new field
at the end? (I know we have to *decode* it in the end, but why want we just shove new entries were they
make sense?)